container_opts.go

```diff
@@ -339,7 +339,11 @@ func WithCDI(s *oci.Spec, annotations map[string]string) NewContainerOpts {
 			return nil
 		}
 
-		registry := cdi.GetRegistry()
+		specDirs := cdiSpecDirs
+		if specDirs == nil {
```

Comment:
Is this strictly necessary? Does `cdi.WithSpecDirs` not handle the `nil` case already?

Reply:
you're right. it's not necessary. removed.